app/models/king.rb

```diff
@@ -1,2 +1,9 @@
 class King < Piece
```

Comment:
I agree. I was trying to push code up so that other people can discuss valid_move? and get going on subclassing move_legal? in the piece subclasses. I will off line until sometime tomorrow.